src/common/url-parser.ts

```diff
@@ -10,6 +10,26 @@ export class UrlParser {
     public areURLHostNamesEqual(urlA: string, urlB: string): boolean {
```

Comment:
Seems like what this method is doing has evolved, and its name should be updated to match, if only to describe the scenario where we want file URLs to compare hrefs but other URLs to compare hosts.

Reply:
Agreed. Will update the name. Any suggestions for it though? `equateURLs` ? or `areURLsEqual`?